docs/vm_builds/README.md

```diff
@@ -0,0 +1,285 @@
+         ],
+         "network_adapters":[
+            {
+               "network":"{{user `vsphere-network`}}",
+               "network_card":"vmxnet3"
+            }
+         ],
+         "password":"{{user `vsphere-password`}}",
+         "storage":[
+            {
+               "disk_size":"{{user `vm-disk-size`}}",
+               "disk_thin_provisioned":true
+            }
+         ],
+         "type":"vsphere-iso",
+         "username":"{{user `vsphere-user`}}",
+         "vcenter_server":"{{user `vsphere-server`}}",
+         "vm_name":"{{user `vm-name`}}",
+         "ssh_password":"{{user `winadmin-password`}}",
+         "ssh_username":"Administrator",
+         "ssh_timeout":"50m"
+      }
+   ],
+   "provisioners":[
+      {
+	"pause_before": "120s",
+	"inline":["ipconfig"],
+        "type":"powershell",
+	"elevated_user": "Administrator",
+        "elevated_password": "{{user `winadmin-password`}}",
+	"max_retries":"10"
+      }
+   ],
+   "sensitive-variables":[
+      "vsphere_password",
+      "winadmin_password"
+   ],
+   "variables":{
+      "os_iso_path":"<Windows_os_iso_path>",
+      "vm-cpu-num":"2",
+      "vm-disk-size":"128000",
+      "vm-mem-size":"4096",
+      "vm-name":"<template_to_be_created>",
+      "vsphere-cluster":"Cluster-1",
+      "vsphere-datacenter":"<datacenter_name>",
+      "vsphere-datastore":"<datastore_name>",
+      "vsphere-folder":"<folder_where_template_gets_built>",
+      "vsphere-network":"network_segment_name",
+      "vsphere-password":"<password>",
+      "vsphere-server":"<server_name>",
+      "vsphere-user":"<user_name>",
+      "winadmin-password":"<Windows_os_password>"
+   }
+}
+```
+
+Please note that the `authorized_keys` file contains the key that should be copied to Windows VM
+
+### Variables
+`<tools_iso_path>` - Path where VMWare tools iso is available
+`<Windows_os_iso_path>` - Path where Windows ISO was dowloaded to
+`<template_to_be_created>` - Name of the Windows vSphere template that will be created by successful packer run
+`<datacenter_name>` - Name of the vSphere datacenter
+`<datastore_name>` - Name of the vSphere datastore
+`<folder_where_template_gets_built>` - Path where Windows template gets built by successful packer run
+`<password>` - vSphere environment password
+`<server_name>` - vSphere environment URL
+`<user_name>` - vSphere username to login
+`<Windows_os_password>` - Password for the Windows template
+
+## What actually happens during build
+
+Packer mounts the Windows iso and starts the VM. 
+- If `floppy_files` section is present, all the files specified there will be copied to the floppy disk of the mounted iso which is represented as `a:\` drive in the Windows VM
+- `Autounattend.xml` is a special file in Windows which gets automatically executed once the VM starts. You can specify all the commands that needs to executed on first boot. We execute some of the files specified in the `scripts` directory during the first boot.
```

Comment:
`We execute some of the files specified in the `scripts` directory during the first boot.` is ambiguous. Please call out the scripts here.